packages/core/src/components/dropdown-button/dropdown-button.tsx

```diff
@@ -0,0 +1,145 @@
+import { Component, h, Prop, Watch, Element } from "@stencil/core";
+import clsx from "clsx";
+import { tabbable } from "tabbable";
+import { v4 as uuidv4 } from "uuid";
+
+function tabInPopup(host: Element, down: boolean) {
+  const tabs = tabbable(host);
+  const currentIndex = tabs.findIndex((e) => e === document.activeElement);
+  const nextIndex = down
+    ? currentIndex >= tabs.length - 1
+      ? 0
+      : currentIndex + 1
+    : currentIndex <= 0
+    ? tabs.length - 1
+    : currentIndex - 1;
+  (tabs[nextIndex] as HTMLElement)?.focus();
+}
+
+function escapePopup(host: Element) {
+  host.shadowRoot?.children[0]?.getElementsByTagName("button")[0]?.focus();
+  host.setAttribute("open", "false");
+}
+
+function closePopupOnLastTab(host: Element) {
+  if (tabbable(host).pop() === document.activeElement) {
+    host.setAttribute("open", "false");
+  }
+}
+
+@Component({
+  tag: "dso-dropdown-button",
+  shadow: true,
+})
+export class DropdownButton {
+  /**
+   * Whether the dropdown is open or closed.
+   * This attribute is reflected and mutable.
+   */
+  @Prop({ reflect: true, mutable: true })
+  open = false;
+
+  /**
+   * The text of the dropdown button.
+   * */
+  @Prop()
+  label: string = "";
+
+  /**
+   * The button style variant
+   */
+  @Prop()
+  variant: "primary" | "secondary" | "tertiary" = "primary";
```

Comment:
Ik stel voor dat we de default variant op `secondary` zetten

Reply:
Is niet meer van toepassing nu de button in in een slot staat